src/pages/library/Datasets.js

```diff
@@ -445,8 +461,8 @@ export const navPaths = [
   {
     name: 'library-datasets',
     path: '/library/datasets',
-    component: Datasets,
-    public: true,
+    component: DatasetsAuthWrapper,
+    public: false,
```

Comment:
If this page is going to require login, should we remove it from the sidebar when the user is not logged in?

![Screen Shot 2022-06-29 at 2 48 48 PM](https://user-images.githubusercontent.com/1156625/176513342-307aa5c5-8ef5-4f2f-a099-2e8ea1b43548.png)

Reply:
the 'View Workspaces' card also requires login